test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -97,7 +97,8 @@ Describe "Stream writer tests" -Tags "CI" {
            }
            else
            {
-               $result.Computer | Should Be $(uname -n)
+               # Use Match instead of Be so we can avoid dealing with a potential domain name
+               $result.Computer | Should Match ".*$(uname -n)"
```

Comment:
I used the existing regex which is working for windows.  I just tested it and didn't think about what it was doing.

Also, we don't want to require it to be an FQDN.  So the regex should be `^$(uname -n)`

I'd stick with `uname`, it's widely available on Linux.  Or switch to `[environment]::MachineName`